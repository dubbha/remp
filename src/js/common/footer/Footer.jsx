import React from 'react';
import Logo from '../logo';
import ErrorBoundary from '../errorBoundary';
import './style.sass';

const Footer = () => (
  <footer className="footer">
    <ErrorBoundary>
      <Logo />
    </ErrorBoundary>
  </footer>
);

export default Footer;
