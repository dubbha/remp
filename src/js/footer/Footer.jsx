import React from 'react';
import Logo from '../logo';
import './style.sass';

const Footer = () => (
  <footer className="footer">
    <Logo />
  </footer>
);

export default Footer;
