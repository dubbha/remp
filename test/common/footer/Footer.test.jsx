import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from 'common/footer';

describe('Footer', () => {
  it('should render successfully', () => {
    const wrapper = shallow(<Footer />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
