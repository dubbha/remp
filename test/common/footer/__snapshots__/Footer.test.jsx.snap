// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render successfully 1`] = `
<footer
  className="footer"
>
  <ErrorBoundary>
    <Logo />
  </ErrorBoundary>
</footer>
`;
